iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -304,9 +304,8 @@ export class Inspector {
             }
 
             // Options
-            this._optionsMenu.addLabel(i18next.t("menu.options"));
-
-            if (this._reportSet.usesCalculations()) {
+            if (this._reportSet.usesCalculations() && !this._iv.isFeatureEnabled(FEATURE_HIDE_CALCULATION_MODE_OPTION)) {
+                this._optionsMenu.addLabel(i18next.t("menu.options"));
```

Comment:
note: typo in the pr description example json (value should be `true`): 

> Add:
>
> ```json
>     "features": {
>         "hide_calculation_mode_option": false
>     },
> ```
> 
> to `ixbrlviewer.config.json`.

Reply:
Oops - fixed. I flipped to `false` to test it the other way in my test config and then copypasted too quickly.